generators/client/templates/angular/src/main/webapp/app/layouts/profiles/page-ribbon.css.ejs

```diff
@@ -22,10 +22,10 @@ Developement Ribbon
 .ribbon {
     background-color: rgba(170, 0, 0, 0.5);
     left: -3.5em;
-    moz-transform: rotate(-45deg);
-    ms-transform: rotate(-45deg);
-    o-transform: rotate(-45deg);
-    webkit-transform: rotate(-45deg);
+    -moz-transform: rotate(-45deg);
```

Comment:
react and sass please

Reply:
No react version of this file. Fixing sass now

Reply:
Fixed and pushed sass file for this one. Confirmed no `.ribbon` specific css for react.

Reply:
you wont find same file, look for occurance of the text

Reply:
Done. No file representing the same issue for a transform css that I see.

Reply:
Also performed a general search for `webkit-`, `moz-`, and `ms-` and all instances are prefixed with the dash necessary to pass the sonar check and be valid.